src/Orleans/Serialization/BuiltInTypes.cs

```diff
@@ -1850,6 +1852,7 @@ internal static void SerializeInvokeMethodRequest(object obj, ISerializationCont
         internal static object DeserializeInvokeMethodRequest(Type expected, IDeserializationContext context)
         {
             int iid = context.StreamReader.ReadInt();
+            ushort iVersion = context.StreamReader.ReadUShort();
```

Comment:
Exactly my point, we shouldn't care about back compat for this value that shouldn't be persisted, only used for messaging.